### ISSUE TITLE
FNX-5168 ⁃ [Bug] Screen reader can't reach other UI items when quick actions sheet is expanded

### ISSUE THREAD

**Issue by eeejay** [mozilla-mobile/fenix#1667]
## Steps to reproduce
With TalkBack enabled:

* Navigate or explore-by-touch to quick actions handle:
![image](https://user-images.githubusercontent.com/5764/56231641-d5ea2000-6033-11e9-9cba-036c7436b53a.png)
* Double tap to expand quick actions sheet:
![image](https://user-images.githubusercontent.com/5764/56231751-12b61700-6034-11e9-9788-b9baa4aece4a.png)
* Try to navigate to other items in the application, either with swiping or explore by touch

### Expected behavior
All visible items should be accessible

### Actual behavior
Nothing, besides the quick actions handle is accessibility focusable.

## Problem
The BottomSheetBehavior [hides all dependent views](https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomsheet/BottomSheetBehavior.java#L1360) when the sheet is expanded. I guess this is meant for modal sheets or ones that obscure the entire app?

The solution is either to do some coordinated layout foo that i don't know, or override setImportantForAccessibility on each of the dependent views and hope we don't have any other needs for them.

Or maybe something else???

**Comment by asadotzler** [comment]
This is still not working. What's the plan here?

**Comment by jyeontaek** [comment]
I can't replicate this problem on my phone. Which devices does this bug occur?

**Comment by colintheshots** [comment]
This bug needs a better description of the problem. So far it sounds like a description of a modal bottom sheet instead of a bug. "Modal bottom sheets are an alternative to inline menus or simple dialogs on mobile and provide room for additional items, longer descriptions, and iconography. They must be dismissed in order to interact with the underlying content." https://material.io/design/components/sheets-bottom.html#usage

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on RC 1.3.0-rc.3 with Xioami Mi8Lite (Android 9).
